charts/csi-baremetal-operator/templates/manager.yaml

```diff
@@ -29,7 +29,7 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 30Mi
+            memory: 100Mi
```

Comment:
Why 30Mi is enough for RKE and vanilla K8s, but not for OpenShift? What is the difference?

Reply:
It looks like that it's a bug in OpenShift 4.6:
- https://bugzilla.redhat.com/show_bug.cgi?id=1904558
- https://github.com/opencontainers/runc/pull/2812

Reply:
Ok, please please comments in the code.

Reply:
Added